examples/entities/example1.json

```diff
@@ -14,7 +14,8 @@
       "function": "ancien ministre du gouvernement cantonais",
       "wkd_id": "NIL",
       "wkpedia_pagename": "NIL",
-      "confidence": "medium"
+      "confidence_ner": 50.45,
```

Comment:
In the entity id of this example, `loc.adm.nat` should be `pers.ind`

Reply:
I think it already is, but I am checking them all again anyway.
```
    {
      "type": "pers.ind",
      "surface": "M. Wou, ancien ministre du gouvernement cantonais",
      "lOffset": 50,
      "rOffset": 99,
```